packages/riipen-ui-docs/src/pages/components/menu/Simple.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import Menu from "@riipen-ui/components/Menu";
+import MenuItem from "@riipen-ui/components/MenuItem";
+import Button from "@riipen-ui/components/Button";
+
+export default function Simple() {
+  const [anchorState, setAnchorState] = React.useState(null);
+  const [indexState, setIndexState] = React.useState(-1);
+
+  const handleClick = event => {
+    setAnchorState(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorState(null);
+  };
+
+  const selectedChange = index => {
+    setIndexState(index);
+  };
+
+  return (
+    <React.Fragment>
+      <Button onClick={handleClick}>HELLO</Button>
```

Comment:
https://www.youtube.com/watch?v=b_ILDFp5DGA

Reply:
https://youtu.be/YQHsXMglC9A?t=80